<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordDlg</class>
 <widget class="QWidget" name="ChangePasswordDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Password</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>234</width>
     <height>135</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="idLabel">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentPWLabel">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Current Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="newPWLabel">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>New Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="confirmPWLabel">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Confirm Password</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="idLineEdit">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="currentPWLineEdit">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="newPWLineEdit">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="confirmPWLineEdit">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
